receiver/collectdreceiver/collectd.go

```diff
@@ -9,17 +9,22 @@ import (
 	"strings"
 	"time"
 
-	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
-	"google.golang.org/protobuf/types/known/timestamppb"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/sanitize"
 )
 
+type TargetMetricType string
+
 const (
-	collectDMetricDerive   = "derive"
-	collectDMetricGauge    = "gauge"
-	collectDMetricCounter  = "counter"
-	collectDMetricAbsolute = "absolute"
+	GaugeMetricType      = TargetMetricType("gauge")
+	CumulativeMetricType = TargetMetricType("cumulative")
```

Comment:
I think the convention is to not export unless we intend users to consume these. Also doesn't look like `GaugeMetricType` is used. Same applies to `TargetMetricType`, which may be unnecessary (derive and counter remain unexported strings for instance).

Reply:
Thanks!
That makes sense. 
I modified the code and not export them.